board/rt1051/crashdump-serial-number/include/crashdump-serial-number/crashdump_serial_number.hpp

```diff
@@ -0,0 +1,17 @@
+// Copyright (c) 2017-2023, Mudita Sp. z.o.o. All rights reserved.
+// For licensing, see https://github.com/mudita/MuditaOS/LICENSE.md
+
+#pragma once
+#include <string>
+namespace crashdump
+{
+    // Purpose of this file is to keep the device serial number in memory.
```

Comment:
This would be better. Also, if it's the whole file's description, it should be placed up - under the copyright notice.

_The purpose of this file is o keep the devices serial number in the memory.
In case of any crash, the serial number could be easily added to a crash dump filename.
It will allow to identify the crash even after it is moved or copied from the device.
setSerialNumber() should be called during system initialization.
getSerialNumber() is called by the crash dump writer during file creation._